core/templates/dev/head/pages/exploration_editor/EditorServices.js

```diff
@@ -629,7 +629,7 @@ oppia.factory('explorationPropertyService', [
       // Validates the given value and returns a boolean stating whether it
       // is valid or not. THIS CAN BE OVERRIDDEN BY SUBCLASSES. The default
       // behavior is to always return true.
-      _isValid: function(value) {
+      _isValid: function() {
```

Comment:
We're moving away from underscores in private method names, please change to `isValid` as well as all instances where this is called. Where they are public we prepend `$scope`.